salt/cert/init.sls

```diff
@@ -16,7 +16,12 @@ include:
   {% endfor %}
 
   # add some extra names the API server could have
-  {% set extra_names = extra_names + ["DNS: api",
+  {% set extra_names = extra_names + ["DNS: kubernetes",
+                                      "DNS: kubernetes.default",
+                                      "DNS: kubernetes.default.svc",
+                                      "DNS: kubernetes.default.svc.cluster",
+                                      "DNS: kubernetes.default.svc.cluster.local",
```

Comment:
"cluster.local" -> is a pillar value, we should pull it from there. Specically, it's the dns:domain pillar.

The issue will be splitting it properly, since it could be replaced with for example "cluster.mycompany.com" - which would need 3 entries instead of two.